ge/org.osate.ge/src/org/osate/ge/internal/ui/editor/AgeDiagramBehavior.java

```diff
@@ -764,13 +765,7 @@ public IFile getFile() {
 	}
 
 	private void updateProjectByUri(final URI uri) {
-		final IPath projectPath = new Path(uri.toPlatformString(true)).uptoSegment(1);
-		final IResource projectResource = ResourcesPlugin.getWorkspace().getRoot().findMember(projectPath);
-		if(!(projectResource instanceof IProject)) {
-			throw new RuntimeException("Unable to determine project");
-		}
-
-		project = (IProject)projectResource;
+		project = ProjectUtil.getProject(uri);
```

Comment:
Remove this. This is not relevant to the branch and the definition of ProjectUtil's getProject does **not** match that in AgeDiagramBehavior.